spec/models/vm_reconfigure_request_spec.rb

```diff
@@ -27,6 +27,19 @@
     end
   end
 
+  context "#my_queue_name" do
+    it "with valid source should have the VM's queue_name_for_ems_operations" do
+      vm_vmware.update(:ems_id => ems_vmware.id)
+      request.update(:options => {:src_ids => [vm_vmware.id]})
+
+      expect(request.my_queue_name).to eq(vm_vmware.queue_name_for_ems_operations)
+    end
+
+    it "with no source should be nil" do
```

Comment:
@gmcculloug @lfu I saw this on the `#my_zone` spec, but is it possible to reconfigure a VM without saying which VM is being reconfigured?  Or maybe I'm mis-understanding the src_ids usage.

Reply:
This case may happen when VM got removed somehow after the request is created.

Reply:
Okay so its not that the src_id won't be there, just the Vm.find_by will return nil.  Doesn't really matter for this PR because that case is handled just want to understand it.